scenario/MainnetBulkerScenario.ts

```diff
@@ -138,7 +138,7 @@ scenario(
     expectApproximately(
       (await stETH.balanceOf(albert.address)).toBigInt(),
       (await wstETH.getStETHByWstETH(exp(1, 18))).toBigInt(),
-      1n
+      2n
```

Comment:
not sure why this started diverging by 2 wei all of a sudden 🤔

Reply:
stETH has a 2 wei rounding issue sometimes https://github.com/lidofinance/lido-dao/issues/442